dask/array/core.py

```diff
@@ -2660,6 +2660,10 @@ def bincount(x, weights=None, minlength=None):
 
     return Array(dsk, name, chunks, dtype)
 
+@wraps(np.digitize)
+def digitize(a, bins, right=False):
+    func = partial(np.digitize, bins=bins, right=right)
+    return elemwise(func, a)
```

Comment:
My apologies, it's called map_blocks on dask.array.  We should consider an alias to avoid this confusion in the future.